Required updates to CommComm Charter to reflect OpenJS Foundation (#526)
nodejs/community-committee: Community-Committee-Charter.md

The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.

## Section 2. Evolution of OpenJS Foundation Governance.

> Most large, complex open source communities have both a business and a technical governance model. In the OpenJS Foundation, technical leadership for projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council. Community leadership is not explicitly owned or delegated. In the Node.js project, community leadership has been delegated to the Community Committee. The OpenJS Foundation's business leadership is instantiated in a Board of Directors (the “Board”). Community activities, such as events, fall outside of the realm of code and documentation that the Node.js project's TSC oversees. The champions of these organizations have very different skillsets, talents, and concerns. The community organizations of Node.js have long been important to the growth and vitality of the project and will be recognized as the Community Committee ("CommComm").

Author: Removes unnecessary bit: 
```suggestion
Most large, complex open source communities have both a business and a technical governance model. Technical leadership for projects within the OpenJS Foundation is delegated to the projects through their project charters by the OpenJS Cross Project Council. Community leadership is not explicitly owned or delegated. In the Node.js project, community leadership has been delegated to the Community Committee. The OpenJS Foundation's business leadership is instantiated in a Board of Directors (the “Board”). Community activities, such as events, fall outside of the realm of code and documentation that the Node.js project's TSC oversees. The champions of these organizations have very different skillsets, talents, and concerns. The community organizations of Node.js have long been important to the growth and vitality of the project and will be recognized as the Community Committee ("CommComm").
```

There is no specific set of requirements or qualifications for CommComm membership beyond these rules. The CommComm may add additional members to the CommComm by a standard CommComm motion and vote. A CommComm member may be removed from the CommComm by voluntary resignation, or by a standard CommComm motion.

Changes to CommComm membership should be posted in the agenda, and may be suggested as any other agenda item.

No more than one-fourth of the CommComm members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CommComm member, or a change of employment by a CommComm member, creates a situation where more than one-fourth of the CommComm membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CommComm members affiliated with the over-represented employer(s).

The CommComm members shall consist of active members of Community Projects.

The CommComm may, at its discretion, invite any number of non-voting Guests to participate in the public portion of CommComm discussions and meetings.

> The CommComm shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CommComm). Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.

Reviewer: nit: Google Hangout On Air is so last week.

Reviewer 2: s/a Google Hangout On Air/Zoom, if editing other details of the changes to the the universe are appropriate. If I were running this, I'd call this change out of scope.

Author: 100% agree on calling it out of scope. I'm don't doubt that in another three years we'll be using a totally different tool as well.

Author: Going to use this:

```suggestion
> The CommComm shall meet regularly using tools that enable participation by the community (through any appropriate means selected by the CommComm). Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
```

CommComm members are expected to regularly participate in CommComm activities.

In the case where an individual CommComm member -- within any three month period -- attends fewer than 25% of the regularly scheduled meetings, does not participate in CommComm discussions, and does not participate in CommComm votes, the member shall be automatically removed from the CommComm. The member may be invited to continue attending CommComm meetings as a Guest.

## Section 4. Responsibilities of the CommComm.

> Subject to such policies as may be set by the Board, the CommComm is responsible for all cultural development and outreach within the Node.js Foundation, including:

Reviewer 3: ```suggestion
Subject to such policies as may be set by the Board, the CommComm is responsible for all cultural development and outreach within the Node.js Project, including:
```

- Outreach to community organizations

> - Documentation of community organizations

Reviewer 4: Why we removed "Inclusivity WG"? Because this WG isn't active anymore?

Author: At the time of the inception of the CommComm, the Inclusivity WG existed and the CommComm's structure was largely built out by folks from that WG. At the time of writing this document, I can only assume it was future-looking based off the current state of things.

There's a few elements of this that don't match now:
- There is no concept of a "WG" in the CommComm
- The Inclusivity WG does not exist in the same state, and I am relatively certain that there will likely not be another WG that brandishes the exact same name (being explicit: this isn't to say there won't be a WG or initiative that aims to address the same problem space) 
- There is no precedent for including specific WGs or Initiatives in a charter, and it actually _limits_ the scope of what we can do rather than expanding that scope

Reviewer 3: I agree with the last point you made there, mentioning specifics in a charter limits the scope of work for CommComm and calls for unnecessary changes.

Reviewer 4: Makes sense 👍

- Cultural direction of Node.js Project
- Overseeing education initiatives for the Node.js Project
- Project governance and process (including this policy)
- Recommendations for building and developing community projects that align with needs of Node.js
- Mediating cultural conflicts between the Node.js project and/or community projects
- Facilitation with external open source projects, select consortiums and other outside groups

> ## Section 5. Node.js Foundation Operations.

Reviewer 3: ```suggestion
## Section 5. Node.js Project Operations.
```

Author: +1

Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one position to be filled. No election is required if there is only one candidate and no objections to the candidate's election. Nominations for organizations that should be represented on the Community Committee will take place in the GitHub repository. A representative for these organizations shall be selected within the organizations by those active in it.

The CommComm will elect from amongst voting CommComm members:
  - a CommComm Chairperson, responsible for preparing agendas and coordinating CommComm meetings

>   - a OpenJS Foundation Cross Project Council (CPC) voting member to represent the CommComm in the OpenJS Foundation

Reviewer: nit: an OpenJS Fou....

Author: Like this? :) 

```suggestion
  - an OpenJS Foundation Cross Project Council (CPC) voting member to represent the CommComm in the OpenJS Foundation
```

Multiple-candidate methods may be reduced to simple election by plurality when there are only two candidates for one position to be filled. No election is required if there is only one candidate and no objections to the candidate's election. Nominations for organizations that should be represented on the Community Committee will take place in the GitHub repository. A representative for these organizations shall be selected within the organizations by those active in it.

> The CommComm will elect from amongst voting CommComm members a CommComm Chairperson to work on building an agenda for CommComm meetings. The CommComm shall hold annual elections to select a CommComm Chairperson; there are no limits on the number of terms a CommComm Chairperson may serve. The same person may not hold both positions at the same time.

Reviewer 5: You should remove `The same person may not hold both positions at the same time.` as there is only one person.

Reviewer 5: link to equivalent section in TSC charter: https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-6-elections

As discussed adding in the text that CommComm elects an OpenJS voting member.

Author: ```suggestion
The CommComm will elect from amongst voting CommComm members a CommComm Chairperson to work on building an agenda for CommComm meetings. The CommComm shall hold annual elections to select a CommComm Chairperson; there are no limits on the number of terms a CommComm Chairperson may serve.
```

Author: Never mind, this is outdated. Updated version should be good. Going to resolve this!

## Section 9. Roles

> OpenJS Foundation git repositories for Community Committee Initiatives are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.

Reviewer 5: Node sure this should be OpenJS Foundation as the repos are in the Node.js org.  This is what is says in the updated TSC charter: 

>The Node.js git repository is maintained by the TSC and additional Collaborators who are added by the TSC on an ongoing basis.

https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-project-roles

Author: How does this sound:

```suggestion
The Node.js Community Committee git repository and various initiative repositories are are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Reviewer 3: Small nitpicks:

```suggestion
The Node.js git repositories for Community Committee and its initiatives are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Author: Tiny additional nit on your enhancement @ahmadawais:

```suggestion
The Node.js git repositories for the Community Committee and its initiatives are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Reviewer 2: Looks good
